composer.json

```diff
@@ -37,7 +37,7 @@
   "require": {
     "php": ">=7.2",
     "ext-json": "*",
-    "doctrine/annotations": "*",
+    "doctrine/annotations": "^1.7",
```

Comment:
This would block interop with frameworks that have set a 1.6 or lower as dep.
And swagger-php doesn't require 1.7 i think.

Reply:
It breaks using PHP 7.4 and running composer with `--prefer-lowest`.
doctrine 1.7 was released August 2019 - I really cannot imagine anyone really relying on older versions